CONTRIBUTING.md

```diff
@@ -1 +1,38 @@
-# Contributing Guidelines
+# Contributing Guidelines - The Forum Helpers Website
+
+We would love your help! Our whole site is open source, and we welcome contributions big and small if they follow these guidelines.
+
+## How to Contribute
+
+*Before* you create a pull request, make sure you read everything below carefully:
+
+1. Your pull request *must* resolve an issue open, unless you are a part of the editors team. If there are no issues open that match what you are trying to do, open a new one and fill out the issue template. If you don't fully complete the issue template, your issue may be closed.
+2. Now that there is an issue open for the thing you would like to implement/fix, you need to wait to be assigned to the issue. By default, [@Accio1](https://github.com/Accio1/) will be assigned to your issue, unless you use the forum helpers template, which [@leahcimto](https://github.com/leahcimto/) would be assigned by default. If you would like to be assigned, comment the following: "@Accio1, could I please be assigned?". Do not comment this if the issue is marked with the `in-progress` label. Now, an editor may assign you, but they also may mark the issue with the `needs-discussion` label if the issue should be discussed.
+3. Now that you have been assigned, you can make a pull request. To do so, fork the repository, make your changes (it is recommended you make a new branch on your fork for these changes), then you can open a pull request by pressing "Compare" on your fork. For more information, see [this](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork), from the GitHub Documentation.
+4. Your pull request is up, now you need to wait for it to be approved. [@Accio1](https://github.com/Accio1/) will be requested for review by default, and others are free to give their suggestions and approval on your pull request. Your pull request must be approved by an editor before being merged.
+5. Now that your pull request is merged, you can delete your branch. by pressing the "delete branch" button on the merged pull request.
```

Comment:
I think this flows better:
5. Now that your pull request is merged, you can delete your branch. Delete your branch by pressing the "delete branch" button on the merged pull request. You can also delete your fork entirely in your forks settings.

Reply:
See above. I think I am just going to remove step 5.

Reply:
Removed